python/CMakeLists.txt

```diff
@@ -4,30 +4,63 @@ include(UseSWIG)
 find_package(PythonLibsNew 2.7 REQUIRED)
 find_package(NumPy REQUIRED)
 
-set(XCFUN_SWIG ${CMAKE_CURRENT_LIST_DIR}/xcfun_swig.i)
+set_source_files_properties(xcfun_swig.i
+  PROPERTIES
+    CPLUSPLUS
+      ON
+    SWIG_FLAGS
+      "-I${CMAKE_BINARY_DIR}/include;-I${PROJECT_SOURCE_DIR}/api"
+  )
+set(CMAKE_SWIG_OUTDIR ${PROJECT_BINARY_DIR}/${PYMOD_INSTALL_FULLDIR})
 
-set_property(SOURCE "${XCFUN_SWIG}" PROPERTY SWIG_FLAGS "-I${CMAKE_BINARY_DIR}/include" 
-                                                        "-I${PROJECT_SOURCE_DIR}/api")
-set_property(SOURCE "${XCFUN_SWIG}" PROPERTY SWIG_OUTDIR ${CMAKE_CURRENT_BINARY_DIR})
+if(CMAKE_VERSION VERSION_LESS "3.8.0")
+  swig_add_module(xcfun_swig python xcfun_swig.i)
+else()
+  swig_add_library(xcfun_swig LANGUAGE python SOURCES xcfun_swig.i)
+endif()
 
-# requires CMake 3.8
-#swig_add_library(xcfun_swig LANGUAGE python SOURCES "${XCFUN_SWIG}")
-swig_add_module(xcfun_swig python "${XCFUN_SWIG}")
+target_link_libraries(${SWIG_MODULE_xcfun_swig_REAL_NAME}
+  PRIVATE
+    ${PYTHON_LIBRARIES}
+    XCFun
+  )
+target_include_directories(${SWIG_MODULE_xcfun_swig_REAL_NAME}
+  PRIVATE
+    ${PYTHON_INCLUDE_DIRS}
+    ${NUMPY_INCLUDE_DIRS}
+  )
+set_target_properties(${SWIG_MODULE_xcfun_swig_REAL_NAME}
+  PROPERTIES
+    MACOSX_RPATH ON
+    SKIP_BUILD_RPATH OFF
+    BUILD_WITH_INSTALL_RPATH OFF
+    INSTALL_RPATH "$ORIGIN/../../../${CMAKE_INSTALL_LIBDIR}"
```

Comment:
The RPATH does work from the build directory, but for me it is not working from the install directory. 

Using the following line here fixes the problem for me:
`INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}"`

Reply:
Can you give me a short example of how you're trying to use the library and the Python bindings from the install directory? I'll test it out locally and also add it to the docs.

Reply:
export PYTHONPATH=$PREFIX/lib/python
    python -c 'import xcfun; print xcfun.xcfun_splash()'

Is that enough? Let me know if you need more information ...

Reply:
I checked again, the problem is only on MacOS (Sorry for not mentioning that before ...). As I understand, $ORIGIN has to be @loader_path on MacOS, if I change this manually it works also on MacOS.

Reply:
OK, I'll found a CMake solution. It **has** to work...

Reply:
I'll debug a bit on Travis.